keyboards/gami_studio/lex60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,41 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _GAMING 1
+#define _FUNCTION 2
+#define _FIRMWARE 3
```

Comment:
I'd recommend using an `enum` here; it makes it so you don't need to change all the definitions if you add, delete, or reorder the layers.

```c
enum layer_names {
    _BASE,
    _GAMING,
    _FUNCTION,
    _FIRMWARE,
};
```

Replacing Lines 7-10 with this block will be identical to the definitions you have already; there's no change in functionality.